### PR TITLE
 modify rocksdb to use boost serialization

### PR DESCRIPTION
**Pull request by bxq2011hust**

### CODE REVIEW

File: libstorage/LevelDBStorage2.h [hunk 1]
```diff
@@ -49,15 +49,16 @@ class LevelDBStorage2 : public Storage
 
 private:
     void processNewEntries(h256 hash, int64_t num,
-        std::shared_ptr<std::map<std::string, Json::Value> > key2value, TableInfo::Ptr tableInfo,
-        Entries::Ptr entries);
+        std::shared_ptr<std::map<std::string, std::vector<std::map<std::string, std::string>>>>
```
Comment (cyjseagull) [inline libstorage/LevelDBStorage2.h]: Recommend `using xxx = std::map<std::string, std::vector<std::map<std::string, std::string>>>;`

File: libstorage/RocksDBStorage.cpp [hunk 2]
```diff
@@ -148,9 +161,10 @@ void RocksDBStorage::setDB(std::shared_ptr<rocksdb::DB> db)
     m_db = db;
 }
 
-void RocksDBStorage::processEntries(h256 hash, int64_t num,
-    std::shared_ptr<std::map<std::string, Json::Value> > key2value, TableInfo::Ptr tableInfo,
-    Entries::Ptr entries)
+void RocksDBStorage::processNewEntries(h256 hash, int64_t num,
+    std::shared_ptr<std::map<std::string, std::vector<std::map<std::string, std::string>>>>
```
Comment (cyjseagull) [inline libstorage/RocksDBStorage.cpp]: recommend `using xxx = std::map<std::string, std::vector<std::map<std::string, std::string>>>`, the same as above.

File: libstorage/TableFactoryPrecompiled.cpp [hunk 3]
```diff
@@ -127,14 +127,21 @@ bytes TableFactoryPrecompiled::call(
             result = e.errorCode();
         }
 
-        out = abi.abiIn("", u256(result));
-        if (g_BCOSConfig.version() < RC2_VERSION)
+        if (result == 1)
```
Comment (cyjseagull) [inline libstorage/TableFactoryPrecompiled.cpp]: This code is a little confuse, please add some comment here.

File: .circleci/config.yml [hunk 4]
```diff
@@ -164,14 +164,19 @@ jobs:
...
+          command: |
+            mkdir -p build && cd build
+            CC=/usr/bin/clang CXX=/usr/bin/clang++ cmake -DSTATIC_BUILD=ON -DCMAKE_BUILD_TYPE=Release ..
+            make -j2
       - run:
           name: Strip
           command: |
             cd build/bin
-            strip fisco-bcos && rm fisco-bcos.tar.gz
+            strip fisco-bcos
```
Comment (cyjseagull) [inline .circleci/config.yml]: TODO: compile a real `Release` version for outside-users to in case of coredump caused by `assert`

Comment (bxq2011hust) [reply .circleci/config.yml]: yes, this modify is try to complete what you want

File: libstorage/TableFactoryPrecompiled.cpp (hunk 3 above)
Comment (bxq2011hust) [reply libstorage/TableFactoryPrecompiled.cpp]: ok

File: libstorage/RocksDBStorage.cpp (hunk 2 above)
Comment (bxq2011hust) [reply libstorage/RocksDBStorage.cpp]: ok

File: libstorage/LevelDBStorage2.h (hunk 1 above)
Comment (bxq2011hust) [reply libstorage/LevelDBStorage2.h]: ok

File: libstorage/Common.h [hunk 5]
```diff
@@ -49,8 +49,8 @@ const char* const SYS_ACCESS_TABLE = "_sys_table_access_";
 const char* const USER_TABLE_PREFIX = "_user_";
 const char* const SYS_BLOCK_2_NONCES = "_sys_block_2_nonces_";
 
-const int CODE_NO_AUTHORIZED = 50000;
-const int CODE_TABLE_NAME_ALREADY_EXIST = 50001;
+const int CODE_NO_AUTHORIZED = -50000;
```
[the remark this answers is not in the document]
Comment (bxq2011hust) [reply libstorage/Common.h]: yes

**Pull request merged**